pkg/chain/event.go

```diff
@@ -1,15 +1,17 @@
-package eth
+package chain
 
 import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
 // BondedECDSAKeepCreatedEvent is an event emitted on a new keep creation.
 type BondedECDSAKeepCreatedEvent struct {
-	KeepAddress     common.Address   // keep contract address
-	Members         []common.Address // keep members addresses
-	HonestThreshold uint64
-	BlockNumber     uint64
+	// KeepAddress          KeepID // keep contract address
+	Keep                 BondedECDSAKeepHandle
+	Members              []KeepMemberID // keep members addresses
+	HonestThreshold      uint64
+	BlockNumber          uint64
+	ThisOperatorIsMember bool
```

Comment:
I like the idea of defining `ThisOperatorIsMember`. 
It changes our previous unwritten rule a little bit as `BondedECDSAKeepCreatedEvent` is no longer a 1:1 representation of the on-chain event as it has some additional fields. 

We also need to remember to set `ThisOperatorIsMember` once creating the event. I'm a little afraid that we will forget to.